test_scripts/UpdateVideoStreamingCapabilities/002_OnAppCapabilityUpdated_from_mobile_to_HMI_capability_parameter_is_missed.lua

```diff
@@ -0,0 +1,42 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0296-Update-video-streaming-capabilities-during-ignition-cycle.md
+--
+-- Description:
+-- Processing OnAppCapabilityUpdated notification from mobile to HMI
+--
+-- Precondition:
+-- 1. SDL and HMI are started
+-- 2. App1 with NAVIGATION appHMIType is registered
```

Comment:
```suggestion
-- 2. App with NAVIGATION appHMIType is registered
```
Please use App without numbers in case app is single in script